Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Driver/AbstractDriver.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver;
+
+/*
+ * This file is part of the Flowpack.ElasticSearch.ContentRepositoryAdaptor package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\ElasticSearchClient;
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\LoggerInterface;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+
+/**
+ * Abstract Fulltext Indexer
```

Comment:
Need a better header, this abstract is used for cross driver requirements, not sure if we really need it.